File: Task 5.vb
Module Module1

    Sub Main()
        Console.WriteLine("Enter String: ")
        Dim input As String = UCase(Console.ReadLine())

        Dim aCount, eCount, iCount, oCount, uCount As Integer

        For count = 1 To Len(input)
            Dim alphabet As Char = Mid(input, count, 1)

            Select Case alphabet
                Case "A"
                    aCount += 1
                Case "E"
                    eCount += 1
                Case "I"
                    iCount += 1
                Case "O"
                    oCount += 1
                Case "U"
                    uCount += 1
            End Select
        Next

        Console.WriteLine("")
        Console.WriteLine("The Vowel A appeared " & aCount & " times in the String.")
        Console.WriteLine("The Vowel E appeared " & eCount & " times in the String.")
        Console.WriteLine("The Vowel I appeared " & iCount & " times in the String.")
        Console.WriteLine("The Vowel O appeared " & oCount & " times in the String.")
        Console.WriteLine("The Vowel U appeared " & uCount & " times in the String.")

        Console.ReadLine()
    End Sub

End Module

File: Python/Task_5.py
String = input("Enter String: ").upper()

vowelCount = [0] * 5
vowels = ["A", "E", "I", "O","U"]

for x in String:
    if x == "A":
        vowelCount[0] +=  1
    if x == "E":
        vowelCount[1] +=  1
    if x == "I":
        vowelCount[2] += 1
    if x == "O":
        vowelCount[3] +=  1
    if x == "O":
        vowelCount[4] +=  1

for i in range(0, 5):
    print(f"The Vowel {vowels[i]} appeared {vowelCount[i]} times in the String.")